<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Light</class>
 <widget class="QDialog" name="Light">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1874</width>
    <height>937</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>901</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_work">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>730</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仓门到工作位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_hard">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>730</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仓门回硬件位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_conut">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>90</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计算区域：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>190</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>center  X：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_radius">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>270</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Radius：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>180</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_X">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>184</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_rad">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>260</y>
     <width>113</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_Y">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>180</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_light">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>360</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>荧光亮度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_light_num">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>430</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cBox_panel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>820</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_set">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>820</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_get_light">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>530</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/confirmbt.png);
</string>
   </property>
   <property name="text">
    <string>   获取亮度</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_backl">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>730</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/btexit.png);</string>
   </property>
   <property name="text">
    <string>   返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_set_xy">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>260</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
